tests/units/feature_extraction/test_data.py

```diff
@@ -0,0 +1,173 @@
+import numpy as np
+import pandas as pd
+
+from tests.fixtures import DataTestCase
+from tsfresh.feature_extraction.data import to_tsdata, LongTsFrameAdapter, WideTsFrameAdapter, TsDictAdapter
+
+TEST_DATA_EXPECTED_TUPLES = \
```

Comment:
In tsfresh, the unittest framework is used extensively, because nearly all test classes inherit from DataTestCase. I agree, that this might deserve some refactoring, but would do that for all tests at once as a separate merge. What do you think?

Reply:
I see we should tackle this at some point, but its not mandatory for now.